assets/source/setup-guide/app/components/SaveSettingsButton.js

```diff
@@ -12,13 +13,43 @@ import {
 	useSettingsDispatch,
 	useCreateNotice,
 } from '../helpers/effects';
+import prepareForTracking from '../helpers/prepare-for-tracking';
 
-const SaveSettingsButton = () => {
+/**
+ * Clicking on "… Save changes" button.
+ *
+ * @event wcadmin_pfw_save_changes_button_click
+ *
+ * @property {boolean} enable_debug_logging Indicates if Enable debug logging option is checked
+ * @property {boolean} enhanced_match_support Indicates if Enhanced Match Support option is checked
+ * @property {boolean} erase_plugin_data Indicates if Erase Plugin Data option is checked
+ * @property {boolean} product_sync_enabled Indicates if Enable Product Sync option is checked
+ * @property {boolean} rich_pins_on_posts Indicates if Add Rich Pins for Posts option is checked
+ * @property {boolean} rich_pins_on_products Indicates if Add Rich Pins for Products option is checked
+ * @property {boolean} save_to_pinterest Indicates if Save to Pinterest option is checked
+ * @property {boolean} track_conversions Indicates if Track Conversion option is checked
+ * @property {string} context The context in which the event is recorded
+ */
+
+/**
+ * Save Settings button component
+ *
+ * @fires wcadmin_pfw_save_changes_button_click with `{ context: 'pinterest_settings', … }`
```

Comment:
It's no longer hard-coded here.
```suggestion
 * @fires wcadmin_pfw_save_changes_button_click with `{ context: view, … }`
```